content/en/blog/_posts/2019-12-02-gardener-project-update.md

```diff
@@ -47,7 +47,7 @@ The essential idea is that so-called **seed** clusters are used to host the
 control planes of end-user clusters (botanically named **shoots**). \
 Gardener provides vanilla Kubernetes clusters as a service independent of the
 underlying infrastructure provider in a homogenous way, utilizing the upstream
-provided `k8s.gcr.io/*` images as open distribution. The project is built
+provided `k8s.gcr.io/*` images as open distribution _(update: `k8s.gcr.io` has since been deprecated in favor of `registry.k8s.io`)._ The project is built
```

Comment:
```suggestion
provided `k8s.gcr.io/*` images as open distribution _(update: `k8s.gcr.io` has been deprecated in favor of `registry.k8s.io`)._ The project is built
```

Reply:
Hi @NitishKumar06,
Updated the suggested changes, PTAL!